src/dashboard/cd-dashboard-children.vue

```diff
@@ -23,10 +69,58 @@
     padding: 0 32px;
     margin-left: auto;
     min-height: 100%;
-    display:flex;
 
     &__header {
-      padding: 16px;
+      margin: 45px 0 16px 0;
+    }
+
+    &__child {
+      margin: 16px 0 16px 0;
+      display: flex;
+      flex-direction: column;
+    }
+
+    &__name {
+      margin: 16px 0 16px 0;
+      display: flex;
+    }
+
+    &__badges {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-evenly;
+      margin: 16px 0 16px 0;
+
+      &-link {
+        margin: 8px 0 0;
+      }
+
+      &-none {
+        margin: 16px 0 16px 0;
+      }
+    }
+
+    &__badge {
+      margin: 16px;
+      width: 105px;
+      height: 100%;
+      text-align: center;
+
+      &-image {
+        height: 100px;
+        width: 100px;
+        margin-bottom: 8px;
+      }
+      &-text {
+        text-align: center;
+      }
+    }
+
+    &__edit-child {
+      font-size: 14px;
+      display: flex;
```

Comment:
This element doesn't need to be `display: flex;` as you're not distributing any of it's contents. `align-self` works fine because this is a child of a flex element.

Reply:
gotcha